index.html

```diff
@@ -8,15 +8,92 @@
     <title>Portfolio</title>
     <link rel="stylesheet" href="assets/styles/style.css">
     <link rel="stylesheet" href="assets/styles/side-menu.css">
+    <link rel="stylesheet" href="assets/styles/details.css">
     <link rel="stylesheet" href="assets/styles/mediaqueries.css">
     <script src="./assets/js/main.js" defer></script>
     <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
 </head>
-
 <body>
+    <section class="backdrop">
+        <div class="modal">
+            <div class="container-modal">
+                <div class="modal-header">
+                    <div class="title">
+                        <h3 class="desc-title"></h3>
+                        <svg width="24" height="24" class="close-btn" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path fill-rule="evenodd" clip-rule="evenodd"
+                                d="M5.29289 5.29289C5.68342 4.90237 6.31658 4.90237 6.70711 5.29289L12 10.5858L17.2929 5.29289C17.6834 4.90237 18.3166 4.90237 18.7071 5.29289C19.0976 5.68342 19.0976 6.31658 18.7071 6.70711L13.4142 12L18.7071 17.2929C19.0976 17.6834 19.0976 18.3166 18.7071 18.7071C18.3166 19.0976 17.6834 19.0976 17.2929 18.7071L12 13.4142L6.70711 18.7071C6.31658 19.0976 5.68342 19.0976 5.29289 18.7071C4.90237 18.3166 4.90237 17.6834 5.29289 17.2929L10.5858 12L5.29289 6.70711C4.90237 6.31658 4.90237 5.68342 5.29289 5.29289Z"
+                                fill="#67798E" />
+                        </svg>
+
+                    </div>
+
+                    <ul>
+                        <li>CANOPY</li>
```

Comment:
- [ ] Hello @DjibrilM, As requested in this milestone, the pop up modal card should be created dynamically using `array of an object` in your JavaScript file and all the hardcoded line of codes in your HTML files should be removed